CODE_OF_CONDUCT.md

```diff
@@ -34,7 +34,7 @@ This Code of Conduct applies both within project spaces and in public spaces whe
 
 ## Enforcement
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at projectmesa-dev@googlegroups.com. The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at [Matrix](https://matrix.to/#/!CAAdpmuJhdNJkxurrp:matrix.org?via=matrix.org). The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```

Comment:
This Matrix link is strange. It should be possible to set up the room so that it has a public human-readable name.

Reply:
Ah, but I think it doesn't make sense to report abuse in public? I think either it has to be a DM to your Matrix account, or keep it via email?

Reply:
@rht Concur. I tried to see if there was a way to set up a private room so admin get any reports but was unsuccessful. So, the question is who gets the DM? @jackiekazil what do you think?

Reply:
It's not possible with a private room, because then the reporter can see past cases, which should be confidential.

Reply:
Changed to my Matrix handle with a DM